pyproject.toml

```diff
@@ -47,6 +47,7 @@ uvicorn = "^0.35.0"
 watchtower = ">=1.0.0"
 attrs = "^25.0.0"
 zstandard = "^0.23.0"
+rpds-py = "^0.24.0"
```

Comment:
why is it added? it's not a dependency of our app, it might be a dependency of a dependency but that will be handled by poetry dependency resolver

Reply:
Hermetic build was failing without it, I will check what we can do about it.